CRITICAL - 2023-10-10 04:03:55 --> Cannot use object of type App\Models\ProductModel as array
in APPPATH\Controllers\ProductController.php on line 62.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\ProductController->delete('7')
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\CodeIgniter4-4.3.6 -blog\\public\\index.php')
CRITICAL - 2023-10-10 04:07:56 --> Call to undefined method CodeIgniter\HTTP\Response::with()
in APPPATH\Controllers\ProductController.php on line 63.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\ProductController->delete('4')
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\CodeIgniter4-4.3.6 -blog\\public\\index.php')
ERROR - 2023-10-10 05:50:00 --> mysqli_sql_exception: Table 'products' already exists in C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php(295): mysqli->query('CREATE TABLE `p...', 0)
#1 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `p...')
#2 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `p...')
#3 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `p...')
#4 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\app\Database\Migrations\2023-10-08-044439_CreateProductTable.php(27): CodeIgniter\Database\Forge->createTable('products')
#5 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(877): App\Database\Migrations\CreateProductTable->up()
#6 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\BaseCommand.php(114): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\spark(97): CodeIgniter\CLI\Console->run()
#14 {main}
ERROR - 2023-10-10 05:50:28 --> mysqli_sql_exception: Table 'products' already exists in C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php(295): mysqli->query('CREATE TABLE `p...', 0)
#1 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `p...')
#2 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `p...')
#3 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `p...')
#4 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\app\Database\Migrations\2023-10-08-044439_CreateProductTable.php(27): CodeIgniter\Database\Forge->createTable('products')
#5 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(877): App\Database\Migrations\CreateProductTable->up()
#6 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\BaseCommand.php(114): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\spark(97): CodeIgniter\CLI\Console->run()
#14 {main}
ERROR - 2023-10-10 05:51:56 --> mysqli_sql_exception: Table 'products' already exists in C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php(295): mysqli->query('CREATE TABLE `p...', 0)
#1 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `p...')
#2 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `p...')
#3 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `p...')
#4 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\app\Database\Migrations\2023-10-08-044439_CreateProductTable.php(27): CodeIgniter\Database\Forge->createTable('products')
#5 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(877): App\Database\Migrations\CreateProductTable->up()
#6 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}
ERROR - 2023-10-10 05:53:49 --> mysqli_sql_exception: Table 'products' already exists in C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php(295): mysqli->query('CREATE TABLE `p...', 0)
#1 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `p...')
#2 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `p...')
#3 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `p...')
#4 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\app\Database\Migrations\2023-10-08-044439_CreateProductTable.php(27): CodeIgniter\Database\Forge->createTable('products')
#5 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(877): App\Database\Migrations\CreateProductTable->up()
#6 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}
ERROR - 2023-10-10 05:54:33 --> mysqli_sql_exception: Table 'products' already exists in C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php(295): mysqli->query('CREATE TABLE `p...', 0)
#1 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `p...')
#2 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `p...')
#3 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `p...')
#4 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\app\Database\Migrations\2023-10-08-044439_CreateProductTable.php(27): CodeIgniter\Database\Forge->createTable('products')
#5 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(877): App\Database\Migrations\CreateProductTable->up()
#6 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\BaseCommand.php(114): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\spark(97): CodeIgniter\CLI\Console->run()
#14 {main}
ERROR - 2023-10-10 05:55:07 --> mysqli_sql_exception: Table 'expances' already exists in C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php(295): mysqli->query('CREATE TABLE `e...', 0)
#1 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `e...')
#2 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `e...')
#3 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `e...')
#4 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\app\Database\Migrations\2023-10-10-032938_CreateExpenceTable.php(26): CodeIgniter\Database\Forge->createTable('expances')
#5 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(877): App\Database\Migrations\CreateExpenceTable->up()
#6 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\BaseCommand.php(114): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\spark(97): CodeIgniter\CLI\Console->run()
#14 {main}
ERROR - 2023-10-10 05:55:46 --> mysqli_sql_exception: Table 'products' already exists in C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php(295): mysqli->query('CREATE TABLE `p...', 0)
#1 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `p...')
#2 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `p...')
#3 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `p...')
#4 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\app\Database\Migrations\2023-10-08-044439_CreateProductTable.php(27): CodeIgniter\Database\Forge->createTable('products')
#5 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(877): App\Database\Migrations\CreateProductTable->up()
#6 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\BaseCommand.php(114): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Commands\Database\MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call('migrate', Array)
#11 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\MigrateRefresh->run(Array)
#12 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate:refresh', Array)
#13 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\spark(97): CodeIgniter\CLI\Console->run()
#14 {main}
ERROR - 2023-10-10 05:56:43 --> mysqli_sql_exception: Table 'products' already exists in C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php(295): mysqli->query('CREATE TABLE `p...', 0)
#1 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('CREATE TABLE `p...')
#2 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('CREATE TABLE `p...')
#3 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\Forge.php(546): CodeIgniter\Database\BaseConnection->query('CREATE TABLE `p...')
#4 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\app\Database\Migrations\2023-10-08-044439_CreateProductTable.php(27): CodeIgniter\Database\Forge->createTable('products')
#5 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(877): App\Database\Migrations\CreateProductTable->up()
#6 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MigrationRunner.php(188): CodeIgniter\Database\MigrationRunner->migrate('up', Object(stdClass))
#7 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}
ERROR - 2023-10-10 06:03:56 --> mysqli_sql_exception: Unknown column 'price' in 'field list' in C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\app\Controllers\ProductController.php(34): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CodeIgniter.php(932): App\Controllers\ProductController->store()
#9 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
#10 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\CodeIgniter4-4.3.6 -blog\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-10-10 06:03:56 --> Unknown column 'price' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `products` (`product`, `category`, `price`, `sku`, `model`) VALUES (\'sdf\', \'sdf\', \'232\', \'wer\', \'sdf\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `products` (`product`, `category`, `price`, `sku`, `model`) VALUES (\'sdf\', \'sdf\', \'232\', \'wer\', \'sdf\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `products` (`product`, `category`, `price`, `sku`, `model`) VALUES (\'sdf\', \'sdf\', \'232\', \'wer\', \'sdf\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `products` (`product`, `category`, `price`, `sku`, `model`) VALUES (:product:, :category:, :price:, :sku:, :model:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\ProductController.php(34): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\ProductController->store()
10 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ProductController))
11 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\CodeIgniter4-4.3.6 -blog\\public\\index.php')
